helper/resource/testing.go

```diff
@@ -810,6 +817,13 @@ type ConfigPlanChecks struct {
 	PostApplyPostRefresh []plancheck.PlanCheck
 }
 
+// ImportPlanChecks defines the different points in an Import TestStep when plan checks can be run.
+type ImportPlanChecks struct {
+	// PreApply runs all plan checks in the slice. This occurs after the plan of an Import test is computed. This slice cannot be populated
+	// with TestStep.PlanOnly, as there is no PreApply plan run with that flag set. All errors by plan checks in this slice are aggregated, reported, and will result in a test failure.
+	PreApply []plancheck.PlanCheck
```

Comment:
🤔 - Wondering if we can find a better name for this field. The naming suggests that we will apply the plan after running the checks.

One of the reason that `ConfigPlanChecks` naming is so awkward is because there were so many places to run plan checks that we basically needed to describe the location of where the plan checks executed (pre-apply, post-apply, etc.)

Here, if we're only going to run the plan checks once, maybe we don't need a separate struct at all? Maybe we could following `ConfigStateChecks` which is just a single slice of state checks that runs once.

Reply:
> I like the last two the most, `PreImport` and `PreApply`, and now I'm leaning back towards `PreApply`

Let's :shipit: with `PreApply` and use the upcoming prerelease as a vehicle for feedback.

Reply:
Sounds good to me 👍🏻